account_analytic_sequence/tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from . import test_analytic_sequence
```

Comment:
a new line?

Reply:
One line between license and import is enough, see https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py

Reply:
ok